web/modules/custom/vote_utility/translations/am.po

```diff
@@ -74,6 +74,9 @@ msgstr "ለመምረጥ ይመዝገቡ @state_name"
 msgid "How to register to vote"
 msgstr "ለመምረጥ እንዴት ነው የሚመዘገቡት"
 
+msgid "How to register and update your registration"
+msgstr "ለመምረጥ እንዴት ነው የሚመዘገቡት"
```

Comment:
I feel like it might be good to add a comment and list which string goes with the registration type...